docsearch/docsearch-config.json.hbs

```diff
@@ -1,113 +0,0 @@
-{
```

Comment:
Why are these .hbs files deleted? Ah, cos the whole docsearch is omitted?

Reply:
Indeed, from what I can see in og sandbox and staging, the `docsearch` folder is not there.